### DISCUSSION TITLE
Getting the raw (unmasked) value from Maska?

### DISCUSSION THREAD

**Discussion by brandonleichty** [beholdr/maska#113] [Q&A] [answered]
I have one last question for the evening 🙂

Let's say I have a form input that takes a credit card like this: 

`<input v-maska data-maska="#### #### #### ####">`

If I were to enter "1234 1234 1234 1234", Maska returns that exact string _with_ the spaces. Is there a good (or easy) way to get the _raw_ value without the added spaces from Maska? 

🙏

**Comment by beholdr** [comment, marked as answer]
You could bind mask result to reactive variable which will contain masked and unmasked value.
Example from docs:

``` html
<script setup>
import { reactive } from "vue"
import { vMaska } from "maska"

const binded = reactive({})
</script>

<template>
  <input v-maska="binded">
  <p v-if="binded.completed">
    Masked value: {{ binded.masked }}
    Unmasked value: {{ binded.unmasked }}
  </p>
</template>
```

**Reply by brandonleichty** [reply]
I see. Thanks @beholdr, I really appreciate it. Could you expand on what `binded.completed` is doing / checking for?

**Reply by beholdr** [reply]
Check for `binded.completed` is totally optional here. It’s just a boolean flag shows that mask is completed (i.e. you have full credit card number filled). `binded.unmasked` will contains card number without spaces.
